shared/src/commonMain/kotlin/com/example/tasky/agenda/domain/model/AgendaItem.kt

```diff
@@ -5,6 +5,11 @@ import com.example.tasky.agenda.data.model.RemoteEvent
 import com.example.tasky.agenda.data.model.RemotePhoto
 import com.example.tasky.agenda.data.model.RemoteReminder
 import com.example.tasky.agenda.data.model.RemoteTask
+import com.example.tasky.database.model.AttendeeSerialized
+import com.example.tasky.database.model.EventEntity
+import com.example.tasky.database.model.PhotoSerialized
+import com.example.tasky.database.model.ReminderEntity
+import com.example.tasky.database.model.TaskEntity
```

Comment:
Mappers from data to domain should be done in your data layer and not your domain if you want to keep the two layers seperate. Here the domain knows about the `Entity` models which is not allowed.

```
// In domain
data class LoginUser(
    val accessToken: String,
    val refreshToken: String,
    val fullName: String,
    val userId: String,
    val accessTokenExpirationTimestamp: Long
)

// In data
data class LoginUserResponse(
    val accessToken: String,
    val refreshToken: String,
    val fullName: String,
    val userId: String,
    val accessTokenExpirationTimestamp: Long
)

// In data (LoginMappers.kt)
fun LoginUserResponse.toLoginUser(): LoginUser {
    return LoginUser(...)
}

// In data (AuthRepositoryImpl.kt)
fun login(email: String, password: String): Result<LoginUser> { // <- Returns domain model
    // ...
    return Result.Success(loginUserResponse.toLoginUser()) // <- mapping happens in data layer
}
```